_posts/2017-11-20-call-for-volunteers.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: post
+title: Call for Volunteers
+---
+
+PyCon AU 2018 will be the eleventh edition of the Australian national Python conference, and is -- as always -- run by a team of volunteers. 
```

Comment:
I thought the first PyCon AU was in 2010? That would make this the 9th.

Ah, just checked the 2017 site says "PyCon Australia 2017 will be the eighth national conference for users "

Reply:
🤦‍♀️ I can't count.